src/domain/header/AccessibilityMenu.js

```diff
@@ -0,0 +1,92 @@
+import PropTypes from 'prop-types';
+import React, { useState } from 'react';
+import { Checkbox, Button } from 'hds-react';
+import without from 'lodash/without';
+import findIndex from 'lodash/findIndex';
+
+import injectT from '../../../app/i18n/injectT';
+import { getFromLocalStorage, storeInLocalStorage } from '../../../app/utils/localStorageUtils';
+
+export const ACCESSIBILITY_LOCAL_STORAGE_KEY = 'varaamo_selectedViewpoints';
+
+const AccessibilityMenu = ({
+  handleClose,
+  locale,
+  t,
+  viewpoints,
+}) => {
+  const [selectedViewpoints, setSelectedViewpoints] = useState(
+    getFromLocalStorage(ACCESSIBILITY_LOCAL_STORAGE_KEY) || [],
+  );
+
+  const handleCheck = viewpoint => (e) => {
+    if (e.target.checked && !selectedViewpoints.includes(viewpoint.viewpointId)) {
+      setSelectedViewpoints(prevState => [...prevState, viewpoint.viewpointId]);
+    }
+
+    if (!e.target.checked) setSelectedViewpoints(without(selectedViewpoints, viewpoint.viewpointId));
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+
+    storeInLocalStorage(ACCESSIBILITY_LOCAL_STORAGE_KEY, selectedViewpoints);
+    handleClose();
+  };
+
+  return (
+    <form className="app-AccessibilityMenu" onSubmit={handleSubmit}>
+      <div className="app-AccessibilityMenu__btn-wrapper">
```

Comment:
> <div className="app-AccessibilityMenu__btn-wrapper">

This suggestion diverges from the designs a bit, but would it make sense to make this top wrapper sticky and remove the bottom one? Right now both submit buttons have the same functionality, so it's a bit confusing to have one called "save settings" and another that just has the text "save".

I think the miro designs specify that the menu doesn't automatically close on save, so this top wrapper would contain the text "changes saved" after click, and possibly reset once the selection has changed, but IMO that's also a bit questionable as closing the menu would then require you to explicitly click "Close" every single time.

Reply:
Yes, I agree with you. I removed the bottom buttons group.
I am not also sure whether to make the "Save" button closes the menu or not, for now, I left as it is 🤷 .

Reply:
Seems fine to me :).